tools/gulp/tasks/examples.ts

```diff
@@ -0,0 +1,52 @@
+import {task, watch} from 'gulp';
+import {join} from 'path';
+import {main as tsc} from '@angular/tsc-wrapped';
+import {SOURCE_ROOT, DIST_BUNDLES, DIST_EXAMPLES} from '../constants';
+import {sequenceTask, sassBuildTask, copyTask, triggerLivereload} from '../util/task_helpers';
+import {createRollupBundle} from '../util/rollup-helper';
+import {transpileFile} from '../util/ts-compiler';
+import {buildModuleEntry, composeRelease} from '../util/package-build';
+import {ScriptTarget, ModuleKind} from 'typescript';
+import {writeFileSync} from 'fs';
```

Comment:
Looks like a lot of unused imports in this file (and library.ts / release.ts) now